### ISSUE TITLE
Colouring the resulting `SVGLayer` apparently produces no effect

### ISSUE THREAD

**Issue by raphaelcruzeiro** [mchoe/SwiftSVG#73]
Colouring the resulting `SVGLayer` apparently produces no effect:

```swift
let image = UIView(SVGNamed: "hollow_pin") { (l: SVGLayer) -> Void in
    l.fillColor = UIColor.blue.cgColor
    l.strokeColor = l.fillColor
}
```

The code above produces an image with a black stroke and clear fill. Am I missing something?

**Comment by mchoe** [comment]
Hi @raphaelcruzeiro, this should work. Can you provide a copy of the SVG file that you're using so I can investigate further?

**Comment by raphaelcruzeiro** [comment]
Hey @mchoe,

This is the SVG I am attempting to render:

```xml
<?xml version="1.0" encoding="utf-8"?>
<!-- Generator: Adobe Illustrator 19.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 22.8 34.2" style="enable-background:new 0 0 22.8 34.2;" xml:space="preserve">
<title>Bike-unselected</title>
<path d="M22.8,11.5L22.8,11.5c0.1-6.3-5-11.5-11.3-11.5S0.1,4.9,0,11.3c0,0,0,0,0,0c0,0.8-0.5,9.4,8.3,21.2c0.1,0.1,0.2,0.2,0.3,0.3
	c1.2,1.6,3.3,1.9,4.9,0.7c0.3-0.2,0.5-0.5,0.7-0.7c0.1-0.1,0.2-0.2,0.3-0.3C22.9,21.3,22.9,13.1,22.8,11.5z M13.2,31.5L13,31.6
	l-0.1,0.1c-0.5,0.8-1.6,1.1-2.5,0.5c-0.2-0.1-0.4-0.3-0.5-0.5l-0.1-0.1l-0.1-0.1l-0.1,0C1,19.8,1.7,11.7,1.7,11.4v-0.1
	c0-5.4,4.3-9.7,9.7-9.7c5.3,0,9.7,4.3,9.7,9.7v0.1C21.2,12.4,21.4,20.4,13.2,31.5L13.2,31.5z"/>
<path d="M11.8,5.3c-0.1,0-0.3,0-0.3,0C8,5.2,5.1,7.9,5,11.3c0,0.1,0,0.2,0,0.3c0,1.4,0.2,2.7,0.7,4c0.3,0.7,2.9,0.5,3.2-0.2
	c0.3-0.7-0.3-2.4-0.3-3.8c-0.2-1.7,1-3,2.6-3c1.6-0.1,3,1.1,3,2.7c0,0.1,0,0.3,0,0.3c0,1.4-0.7,3.2-0.3,3.8c0.3,0.7,2.9,0.9,3.2,0.2
	c0.4-1.3,0.7-2.6,0.7-4C17.9,8.3,15.2,5.4,11.8,5.3z M6.4,10.4c0-0.3,0.2-0.5,0.5-0.5s0.5,0.2,0.5,0.5S7.2,11,6.9,11
	S6.4,10.7,6.4,10.4z M7.1,13.7c-0.3,0-0.5-0.2-0.5-0.5c0-0.3,0.2-0.5,0.5-0.5c0.3,0,0.5,0.2,0.5,0.5C7.6,13.4,7.3,13.7,7.1,13.7z
	 M8.4,8.5C8.2,8.5,7.9,8.2,7.9,8c0-0.3,0.2-0.5,0.5-0.5C8.7,7.4,9,7.7,9,8C9,8.2,8.7,8.5,8.4,8.5z M11.5,7.4c-0.3,0-0.5-0.2-0.5-0.5
	s0.2-0.5,0.5-0.5S12,6.6,12,6.9S11.8,7.4,11.5,7.4z M14.4,8.5c-0.3,0-0.5-0.2-0.5-0.5c0-0.3,0.2-0.5,0.5-0.5c0.3,0,0.5,0.2,0.5,0.5
	C14.9,8.2,14.7,8.5,14.4,8.5z M15.8,13.7c-0.3,0-0.5-0.2-0.5-0.5c0-0.3,0.2-0.5,0.5-0.5c0.3,0,0.5,0.2,0.5,0.5
	C16.3,13.4,16.1,13.7,15.8,13.7z M15.9,11c-0.3,0-0.5-0.2-0.5-0.5s0.2-0.5,0.5-0.5c0.3,0,0.5,0.2,0.5,0.5S16.2,11,15.9,11z"/>
</svg>
```

No matter what I do, the path is always black. Please let me know if I can provide more info.

**Comment by mchoe** [comment]
Hi @raphaelcruzeiro, I'm not sure whether this was fixed recently, but as of `v2.1.5` this seems to be working. I added this to the iOS example app and you can checkout the latest commit on `master` to see this in action. Closing for now, but let me know if you continue to have issues.
